inspirehep/modules/workflows/actions/merge_approval.py

```diff
@@ -32,4 +34,16 @@ class MergeApproval(object):
     @staticmethod
     def resolve(obj, *args, **kwargs):
         """Resolve the action taken in the approval action."""
-        pass
+
+        obj.extra_data["approved"] = True
```

Comment:
So we are sure that it's always approved?

Reply:
For now yes, but in the future, of course, we can have the chance to reject the merge.